### DISCUSSION TITLE
[Feature] Option to allow guests to modify description and or tags

### DISCUSSION THREAD

**Discussion by grasponcrypto** [immich-app/immich#3990] [Feature Request]
### The feature

I'm creating a shared album which I would like to share with friends for a reunion, and would like to be able to allow anyone with the url to also add tags and/or modify the description of any photo in the album.  

The idea is that one should be able to find a picture with themselves in it, and add a description giving context to the picture (since these pictures are decades old).

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by diabWH3** [comment]
Seconded. Just inherited a shoe box full of old film prints without any dates, notes, or names of the people in the photos. Would love to share an album of these photos to a group of friends and crowd-source relevant info about each photo.
